docs/interactions/Slash_Commands.md

```diff
@@ -0,0 +1,841 @@
+| data?\*    | ApplicationCommandInteractionData | the command data payload                                |
+| guild_id   | snowflake                         | the guild it was sent from                              |
+| channel_id | snowflake                         | the channel it was sent from                            |
+| member     | GuildMember                       | guild member data for the invoking user                 |
+| token      | string                            | a contintuation token for responding to the interaction |
+| version    | int                               | read-only property, always `1`                          |
+
+\* This is always present on `ApplicationCommand` interaction types. It is optional for future-proofing against new interaction types
+
+###### InteractionType
+
+| Name               | Value |
+|--------------------|-------|
+| Ping               | 1     |
+| ApplicationCommand | 2     |
+
+###### ApplicationCommandInteractionData
+
+| Field    | Type                                             | Description                       |
+|----------|--------------------------------------------------|-----------------------------------|
+| id       | snowflake                                        | the ID of the invoked command     |
+| name     | string                                           | the name of the invoked command   |
+| options? | array of ApplicationCommandInteractionDataOption | the params + values from the user |
+
+###### ApplicationCommandInteractionDataOption
+
+All options have names, and an option can either be a parameter and input value--in which case `value` will be set--or it can denote a subcommand or group--in which case it will contain a top-level key and another array of `options`. 
+
+`value` and `options` are mututally exclusive.
+
+| Field    | Type                                             | Description                                     |
+|----------|--------------------------------------------------|-------------------------------------------------|
+| name     | string                                           | the name of the parameter                       |
+| value?   | OptionType                                       | the value of the pair                           |
+| options? | array of ApplicationCommandInteractionDataOption | present if this option is a group or subcommand |
+
+###### Interaction Response
+
+After receiving an interaction, you must respond to acknowledge it. This may be a `pong` for a `ping`, a message, or simply an acknowledgement that you have received it and will handle the command async.
+
+Interaction responses may choose to "eat" the user's command input if you do not wish to have their slash command show up as message in chat. This may be helpful for slash commands, or commands whose responses are asynchronous or ephemeral messages.
```

Comment:
I feel "swallow" is the more widely used term here, but "eat" is a fun word.

Reply:
Yknow, I _knew_ "eat" wasn't quite it but I knew it was some sort of eating-based word. I'm keeping it now though I've become attached :p